stacks-signer/src/runloop.rs

```diff
@@ -365,10 +402,15 @@ impl<C: Coordinator> RunLoop<C> {
     fn handle_proposed_blocks(&mut self, blocks: Vec<NakamotoBlock>) {
         for block in blocks {
             // Store the block in our cache
-            self.blocks.insert(
-                block.header.signer_signature_hash(),
-                BlockInfo::new(block.clone()),
-            );
+            // self.blocks.insert(
+            //     block.header.signer_signature_hash(),
+            //     BlockInfo::new(block.clone()),
+            // );
+
+            println!("Inserting into DB in handle_proposed_blocks: {:?}", block);
```

Comment:
Before converting to a full PR, please remove all `println!()`s and use the logging facilities.  Thanks!